Pages/ManageIndividualVolunteer.razor

```diff
@@ -0,0 +1,106 @@
+﻿@page "/volunteer/{Id:guid}"
+@using System.Linq
+@using Karma.Models
+@inject IJSRuntime m_jsRuntime
+@using Microsoft.EntityFrameworkCore;
+
+@if (m_httpContextAccessor.HttpContext.User.Identity.IsAuthenticated)
+{
+    @if (volunteer != null)
+    {
+        <h1>Manage volunteer</h1>
+
+        <label for="fname">Volunteer name:</label>
+        <input type="text" @bind="@volunteer.Name" @oninput="@((e) => { volunteer.Name = (string)e.Value; })" />
+        <br>
+        <br>
+        <label for="fname">Volunterr surname:</label>
+        <input type="text" @bind="@volunteer.Surname" @oninput="@((e) => { volunteer.Surname = (string)e.Value; })" />
+        <br>
+        <br>
+        <br>
```

Comment:
Use with a closing tag. Instead of <br> it should be <br /> I don't know which part of our project(maybe browswer) understands that you meant <br /> but let's not count on this 'understanding'

Reply:
![image](https://user-images.githubusercontent.com/76516162/136164415-a07d6698-4d1d-4b76-b7dd-dd61c92f83ea.png)